documentation/database/big-query.md

```diff
@@ -61,15 +61,20 @@ Support Level determines the degree of support available for this database ([lea
 
 ### Performance
 
-GCP BigQuery can suffer from performance issues while executing schema changes.
+Executing multiple schema changes against GCP BigQuery is comparatively slow as it is optimized for data. You can read more about it [here](https://cloud.google.com/blog/products/data-analytics/troubleshoot-bigquery-performance-with-these-dashboards).
 
-Flyway Teams Edition solves this via [batching](/documentation/configuration/parameters/batch) which combines schema changes to reduce the network overhead and improves performance.
+Flyway Teams Edition alleviates this via batching which executes multiple schema changes in one batch to minimize latency and improve performance.
```

Comment:
Don't like the term 'batching' being defined by the term 'batch'. Seems a little redundant. Could this be better said as 'via batching which executes multiple schema changes in one **request**'?

Reply:
see latest commit